airflow/www/extensions/init_security.py

```diff
@@ -70,5 +69,4 @@ def init_check_user_active(app):
     @app.before_request
     def check_user_active():
         if get_auth_manager().is_logged_in() and not g.user.is_active:
-            logout_user()
-            return redirect(get_auth_manager().get_url_login())
+            return redirect(get_auth_manager().get_url_logout())
```

Comment:
This change doesn’t looks right?

Reply:
But this is the right change (at least to me). Explanations: the previous code is actually a copy paste from Flask_appbuilder code, see [here](https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/security/views.py#L499). Instead of copying the code from Flask, let's just redirect the user to the logout page and Flask will handle it. Plus, we do not want to expose a method like `logout_user()`in `base_auth_manager`. Some auth manager might not have an api to logout an user but only an url to redirect users to in order to log them out

Reply:
That makes sense @uranusjr ?

Reply:
Hmm, OK. This brings out a more fundamental problem of [using GET for logout](https://stackoverflow.com/questions/3521290/logout-get-or-post) (which `redirect` implicitly means), but I checked the code base and it seems this is already done elsewhere anyway. I hope we’ll change that at some point, but that’s probably nowhere near top priority (nobody seems to have complained yet), and again, this is already done elsewhere, so this change relying on the same behaviour is OK.

Reply:
Interesting article.  It makes sense, and something I wasn't aware of before.  Thanks for sharing it.